config/zones/HU.yaml

```diff
@@ -55,20 +139,47 @@ capacity:
     - datetime: '2024-10-01'
       source: mavir.hu
       value: 3964.8
+    - datetime: '2025-01-01'
+      source: entsoe.eu
+      value: 7473.0
```

Comment:
A change to this value this large is _sus_ to me, even though ENTSO-E also shows the same. Maybe it includes household-size (<=50kW) installations too now?

Maybe there has been a change in how metrics are summarized, but unless the live production data will also include these, things could get confusing.

Reply:
What do you think, @VIKTORVAV99?

Reply:
There really is no way to tell from the data as far as I can see so our bet bet would be to reach out to ENTSOE about it, it's not a priority for me at the moment though as there are bigger data issues to deal with at the moment.